addons/mail/static/src/core/common/message.xml

```diff
@@ -111,7 +113,7 @@
                                             </div>
                                         </t>
                                     </t>
-                                    <t t-if="message.hasTextContent and !env.inChatWindow" t-call="mail.Message.actions"/>
+                                    <t t-if="!message.isPending and message.hasTextContent and !env.inChatWindow" t-call="mail.Message.actions"/>
```

Comment:
There is another call to `mail.Message.actions` below `AttachmentList` and `LinkPreviewList`. You might want to guard it too.